integration_tests/test_mesh_lens_correction.py

```diff
@@ -0,0 +1,98 @@
+    }
+
+    return tform
+
+def test_mesh_lens_correction(render, example_tform_dict, tmpdir_factory):
+    outdir = str(tmpdir_factory.mktemp("mesh_lens"))
+    out_html_dir = outdir
+    example['render'] = render_params
+    example['metafile'] = os.path.join(TEST_DATA_ROOT, 
+                                       "em_modules_test_data", 
+                                       "mesh_lens_correction", 
+                                       "mesh_lens_metafile.json")
+    example['output_dir'] = outdir
+    example['out_html_dir'] = out_html_dir
+    example['z_index'] = 100
+
+    mod = MeshLensCorrection(input_data=example, args=['--output_json', 'mesh_lens_out.json'])
+    mod.run()
+    
+    with open('mesh_lens_out.json', 'r') as f:
+        js = json.load(f)
+
+    with open(js['output_json'], 'r') as f:
+        new_tform_dict = json.load(f)
+
+    with open(js['qc_json'], 'r') as f:
+        qc = json.load(f)
+
+    assert(not qc['hole_sections'] and not qc['gap_sections'])
+
+    #example_tform = renderapi.transform.ThinPlateSplineTransform(dataString=example_tform_dict['dataString'])
+    #new_tform = renderapi.transform.ThinPlateSplineTransform(dataString=new_tform_dict['dataString'])
```

Comment:
How difficult is it to add a residual test comparison based on e.g. a grid?

Reply:
yes planning to add this in next PR

Reply:
I added issue #119 as a reminder so I feel okay about merging this.